internal/adapter/inbound/http/document_handler_test.go

```
@@ -2351,3 +2356,333 @@ func TestDocumentHandler_Create_OversizedFieldRejected(t *testing.T) {
+	r.ServeHTTP(rr, req)
+
+	if rr.Code != http.StatusCreated {
+		t.Fatalf("status = %d, want 201 for skip=false after file, body: %s", rr.Code, rr.Body.String())
+	}
+}
+
+// --- 013: by-reference lookup + PATCH move (re-attribute) ---
+
+// withURLParam attaches a chi route context carrying a single URL param, so a
+// handler that reads chi.URLParam can be invoked directly without a router.
+func withURLParam(req *http.Request, key, value string) *http.Request {
```

golangci-lint (2.12.2)

[error] withURLParam - key always receives "id"

(unparam)